examples/with-couchbase/pages/index.js

```diff
@@ -0,0 +1,254 @@
+import Head from 'next/head'
+import { connectToDatabase } from '../util/couchbase'
+
+export default function Home({ isConnected }) {
+  return (
+    <div className="container">
+      <Head>
+        <title>Create Next App</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>
+        <h1 className="title">
+          Welcome to <a href="https://nextjs.org">Next.js with Couchbase!</a>
+        </h1>
+
+        {isConnected ? (
+          <h2 className="subtitle green">You are connected to Couchbase</h2>
+        ) : (
+          <>
+            <h2 className="subtitle red">
+              You are NOT connected to Couchbase. Try refreshing the page, and
+              if this error persists check the <code>README.md</code> for
+              instructions.
+            </h2>
+            <em className="center">
+              Note: if the database was recently started, you might have to
+              re-start the app (in dev mode) or re-deploy to your serverless
+              environment for changes to take effect.
+            </em>
+          </>
+        )}
+
+        <p className="description">
+          Get started by editing <code>pages/index.js</code>
+        </p>
+      </main>
+
+      <footer>
+        <a
+          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Powered by{' '}
+          <img src="/vercel.svg" alt="Vercel Logo" className="logo" />
+        </a>
+      </footer>
+
+      <style jsx>{`
```

Comment:
Can this use CSS Modules instead of styled-jsx like what create-next-app ships with? Thanks!